src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/Models/InviteOrganizationUsersRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Bit.Core.AdminConsole.Models.Business;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.OrganizationUsers.InviteUsers.Models;
+
+public class InviteOrganizationUsersRequest
+{
+    public OrganizationUserInvite[] Invites { get; } = [];
+    public OrganizationDto Organization { get; }
+    public Guid PerformedBy { get; }
+    public DateTimeOffset PerformedAt { get; }
+
+    public InviteOrganizationUsersRequest(OrganizationUserInvite[] Invites,
+        OrganizationDto Organization,
+        Guid PerformedBy,
+        DateTimeOffset PerformedAt)
+    {
+        this.Invites = Invites;
+        this.Organization = Organization;
+        this.PerformedBy = PerformedBy;
+        this.PerformedAt = PerformedAt;
+    }
+
+    public static InviteOrganizationUsersRequest Create(InviteScimOrganizationUserRequest request) =>
+        new([OrganizationUserInvite.Create(request.Invite, request.ExternalId)],
+            request.Organization,
+            Guid.Empty,
+            request.PerformedAt);
```

Comment:
I may not be understanding why these static methods are better than constructors
```suggestion
    public InviteOrganizationUsersRequest(InviteScimOrganizationUserRequest request)
        : this(
            Invites: [OrganizationUserInvite.Create(request.Invite, request.ExternalId)],
            Organization: request.Organization,
            PerformedBy: Guid.Empty,
            PerformedAt: request.PerformedAt)
    {
    }
```

Reply:
I prefer the way the `Create` static method reads in code. Especially when working with `Func`. It also allows you to put some validation logic into the method, which I prefer over putting in the constructor.

I can switch over to constructors since that seems to be others' preference.

Reply:
That is fair, I can see why you don't want logic inside the constructor. But it's not very intuitive that to create an object you have to use the `Create` method over the constructor.